modules/appeals_api/spec/support/rswag_config.rb

```diff
@@ -961,7 +960,60 @@ def decision_reviews_sc_response_schemas
     }
   end
 
-  def sc_response_schemas = decision_reviews_sc_response_schemas
+  def sc_response_schemas
+    decision_reviews_sc_response_schemas.merge(
+      {
+        'scCreateResponse': {
+          'description': 'Successful response of a 200995 form submission',
+          'type': 'object',
+          'properties': {
+            'data': {
+              'properties': {
+                'id': {
+                  'type': 'string',
+                  'description': 'Unique ID of created supplemental claim',
+                  'example': '97751cb6-d06d-4179-87f6-75e3fc9d875c'
+                },
+                'type': {
+                  'type': 'string',
+                  'description': 'Type of record',
+                  'example': 'supplementalClaim'
+                },
+                'attributes': {
+                  'type': 'object',
+                  'properties': {
+                    'status': {
+                      'type': 'string',
+                      'description': 'Status of created supplemental claim',
+                      'example': AppealsApi::SupplementalClaim::STATUSES.first,
+                      'enum': AppealsApi::SupplementalClaim::STATUSES
+                    },
+                    'createdAt': {
+                      'type': 'string',
+                      'description': 'Created timestamp of the supplemental claim',
+                      'example': '2020-12-16T19:52:23.909Z'
+                    },
+                    'updatedAt': {
+                      'type': 'string',
+                      'description': 'Updated timestamp of the supplemental claim',
+                      'example': '2020-12-16T19:52:23.909Z'
+                    }
+                  }
+                },
+                'formData': { '$ref': '#/components/schemas/scCreate' }
+              }
+            },
+            'included': {
+              'type': 'array',
+              'items': {
+                '$ref': '#/components/schemas/appealableIssue'
```

Comment:
This was only added to for the `'$ref': '#/components/schemas/appealableIssue'` line, which is necessary now that we're actually using appealable issues schemas here